lua/core/event.lua

```diff
@@ -151,7 +151,10 @@ function autocmd.load_autocmds()
 			},
 		},
 	}
-
+	local ok, user_definitions = pcall(require, "user.event")
+	if ok then
+		definitions = vim.tbl_deep_extend("force", definitions, user_definitions)
+	end
```

Comment:
```suggestion
	local ok, user_events = pcall(require, "user.event")
	if ok then
		definitions = vim.tbl_deep_extend("force", definitions, user_events)
	end
```
Better to follow naming conventions here.

Reply:
Resolved below